contracts/src/v0.8/dev/ocr2dr/OCR2DRRegistry.sol

```diff
@@ -88,17 +91,18 @@ contract OCR2DRRegistry is
 
   mapping(address => uint96) /* oracle */ /* LINK balance */
     private s_withdrawableTokens;
+  // The DON address is not used as there is only 1 DON per registry
   struct Commitment {
     uint64 subscriptionId;
     address client;
     uint32 gasLimit;
     uint256 gasPrice;
-    address don;
     uint96 donFee;
     uint96 registryFee;
     uint96 estimatedCost;
     uint256 timestamp;
   }
+  // Only the Commitment hash should be stored on-chain for gas efficiency, however this is not essential for PoC.
```

Comment:
There is already a Jira ticket open for this. I'd prefer to keep comments like this out of the code.